requirements/common.txt

```diff
@@ -1,3 +1,4 @@
 psycopg2
 SQLAlchemy
 numpy
+sparqlwrapper
```

Comment:
I don't think this should be here.
common.txt should only contain the requirements for the package, not the tutorial.
We shouldn't introduce a dependency on sparqlwrapper if it's not needed to use the package.